2/src/components/__tests__/List.spec.js

```diff
@@ -0,0 +1,28 @@
+import { shallowMount } from '@vue/test-utils'
+import List from '../List.vue'
+import ListItem from '../ListItem.vue'
+
+const factory = () => {
+	return shallowMount(List)
+}
+
+describe('List', () => {
+	const wrapper = factory();
+
+	it('Renders a add todo button', () => {
+		const addButton = wrapper.find(".add-button");
+		expect(addButton.text()).toBe("Add todo");
+	})
+
+	it('Renders the correct number of list items', () => {
+		const listItems = wrapper.findAll(ListItem);
+		expect(listItems.length).toBe(3);
+	})
+
+	it('Add a todo after add button was clicked', () => {
```

Comment:
If you use `it` please use simple present and make it a readable sentence: 
```suggestion
	it('adds a todo when add button is clicked', () => {
```